src/openseadragon.js

```diff
@@ -1339,12 +1349,39 @@ function OpenSeadragon( options ){
             flipped:                    false,
 
             // APPEARANCE
-            opacity:                           1,
-            preload:                           false,
-            compositeOperation:                null,
-            imageSmoothingEnabled:             true,
-            placeholderFillStyle:              null,
-            subPixelRoundingForTransparency:   null,
+            opacity:                           1, // to be passed into each TiledImage
+            compositeOperation:                null, // to be passed into each TiledImage
+
+            // DRAWER SETTINGS
+            drawer:                            ['webgl', 'canvas', 'html'], // prefer using webgl, then canvas (i.e. context2d), then fallback to html
+                /**
+                 * drawerOptions dictionary.
```

Comment:
I'm wondering why this is documented here rather than up in the options. I guess it's fine either way, but I feel like often we document such things up in the options. I'm wondering if there's a good principle for where to put it.

Reply:
Moved it away from here towards where `GestureSettings` are defined, since it seems similar to that level of nested options.

Reply:
That's a good solution 🙂